### ISSUE TITLE
Cannot read property 'uri' of undefined

### ISSUE THREAD

**Issue by Spown** [Microsoft/vscode-npm-scripts#46]
```
shell.ts:454 [eg2.vscode-npm-script] Cannot read property 'uri' of undefined: TypeError: Cannot read property 'uri' of undefined
	at Promise (C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:189:77)
	at Promise (<anonymous>)
	at C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:188:16
	at Generator.next (<anonymous>)
	at C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:7:71
	at Promise (<anonymous>)
	at __awaiter (C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:3:12)
	at isYarnManaged (C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:187:12)
	at C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:164:54
	at Generator.next (<anonymous>)
	at C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:7:71
	at Promise (<anonymous>)
	at __awaiter (C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:3:12)
	at validateDocument (C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:161:12)
	at vscode_1.workspace.onDidSaveTextDocument.document (C:\Users\XXXXX\.vscode\extensions\eg2.vscode-npm-script-0.3.2\out\src\main.js:141:13)
	at e.invoke (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:28:57853)
```

**Comment by chrisdias** [comment]
i'm seeing this too

**Comment by egamma** [comment]
Fix published in version 0.3.3
